addons/binding/org.openhab.binding.nest/src/main/java/org/openhab/binding/nest/config/NestBridgeConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package org.openhab.binding.nest.config;
+
+public class NestBridgeConfiguration {
```

Comment:
Please add a JavaDoc with `@author` tag containing a real name of the author. See [Code Style](http://docs.openhab.org/developers/development/guidelines.html#a-code-style).